backend/src/main/java/com/salat/briene/controlleradvices/UserAdvice.java

```diff
@@ -0,0 +1,32 @@
+package com.salat.briene.controlleradvices;
+
+import com.salat.briene.exceptions.AnonymousUserException;
+import com.salat.briene.exceptions.DuplicatedUserException;
+import com.salat.briene.exceptions.UserNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class UserAdvice {
+    @ExceptionHandler(AnonymousUserException.class)
+    public ResponseEntity<?> handleAnonymousUser(AnonymousUserException e) {
+        e.printStackTrace();
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(e.getMessage());
+    }
+
+    @ExceptionHandler(DuplicatedUserException.class)
+    public ResponseEntity<?> handleDuplicatedUser(DuplicatedUserException e) {
+        e.printStackTrace();
+        return ResponseEntity.status(HttpStatus.SEE_OTHER).body(e.getMessage());
```

Comment:
I think **422 Unprocessable Entity** will be more accurate here. 303 should be used if we redirecting to already created entity. But it's not our case. 422 code means that request was understood by server, but it can't be served for some logical reason.